teachertool/src/global.scss

```diff
@@ -92,6 +92,33 @@ code {
     cursor: default;
 }
 
+/*******************************/
+/********** SCROLLBAR  *********/
+/*******************************/
+
+// Modeled after the scrollbar in the MakeCode blocks editor
+
+::-webkit-scrollbar {
+    width: 14px;
+    height: 14px;
+    background-color: rgba(0, 0, 0, 0);
+}
+
+::-webkit-scrollbar-thumb {
+    height: 0;
+    border: 2px solid rgba(0, 0, 0, 0);
+    border-radius: 99px;
+    background-clip: padding-box;
+    background-color: rgb(206, 205, 206);
+    &:hover {
+        // Style matching the standalone editor
+        //background-color: rgb(187, 187, 187);
```

Comment:
Do we want to keep these commented styles?

Reply:
Yes, I'd like to keep them because I don't know why the embedded editor's scrollbars look slightly different. If they ever end up updating to look like the standalone editor, we can just uncomment these lines. It took me longer than I'd like to admit to figure out the border/inset solution. :)